RecoTracker/IterativeTracking/python/HighPtTripletStep_cff.py

```diff
@@ -244,18 +244,27 @@
 fastSim.toModify(highPtTripletStepTracks,TTRHBuilder = 'WithoutRefit')
 
 # Final selection
-from RecoTracker.FinalTrackSelectors.TrackMVAClassifierPrompt_cfi import *
-highPtTripletStep = TrackMVAClassifierPrompt.clone(
-    src	= 'highPtTripletStepTracks',
-    mva = dict(GBRForestLabel = 'MVASelectorHighPtTripletStep_Phase1'),
-    qualityCuts	= [0.2,0.3,0.4],
+from RecoTracker.FinalTrackSelectors.TrackLwtnnClassifier_cfi import *
+from RecoTracker.FinalTrackSelectors.trackSelectionLwtnn_cfi import *
+highPtTripletStep = TrackLwtnnClassifier.clone(
+    src = 'highPtTripletStepTracks',
+    qualityCuts = [0.75, 0.775, 0.8],
 )
 fastSim.toModify(highPtTripletStep,vertices = "firstStepPrimaryVerticesBeforeMixing")
-highBetaStar_2018.toModify(highPtTripletStep,qualityCuts = [-0.2,0.3,0.4])
-pp_on_AA_2018.toModify(highPtTripletStep, 
-        mva = dict(GBRForestLabel = 'HIMVASelectorHighPtTripletStep_Phase1'),
-        qualityCuts = [-0.9, -0.3, 0.85],
-)
+
+from RecoTracker.FinalTrackSelectors.TrackMVAClassifierPrompt_cfi import *
+
+highBetaStar_2018.toReplaceWith(highPtTripletStep, TrackMVAClassifierPrompt.clone(
+     qualityCuts = [-0.2,0.3,0.4],
+     src = 'highPtTripletStepTracks',
+     mva = dict(GBRForestLabel = 'MVASelectorDetachedTripletStep_Phase1') 
```

Comment:
this was apparently a bug in this PR: the mva forest label is wrong.
Apparently nobody noticed.